### DISCUSSION TITLE
Is it possible to update the previous version (2.1.0) to the new one (2.2.0) without doing a new installation, or is it necessary to reinstall?

### DISCUSSION THREAD

**Discussion by douglasdock** [Joshua-Riek/ubuntu-rockchip#868] [General]
Is it possible to update the previous version (2.1.0) to the new one (2.2.0) without doing a new installation, or is it necessary to reinstall?

**Comment by Joshua-Riek** [comment]
All changes can be received from APT. So there is no need to install again.

**Reply by douglasdock** [reply]
Friend, thank you very much for your work and dedication.
